custom_components/adaptive_lighting/const.py

```diff
@@ -111,6 +114,9 @@ def int_between(min_int, max_int):
     (CONF_SEPARATE_TURN_ON_COMMANDS, DEFAULT_SEPARATE_TURN_ON_COMMANDS, bool),
     (CONF_SEND_SPLIT_DELAY, DEFAULT_SEND_SPLIT_DELAY, int_between(0, 10000)),
     (CONF_ADAPT_DELAY, DEFAULT_ADAPT_DELAY, int_between(0, 10000)),
+    (CONF_STRICT_ADAPTING, DEFAULT_STRICT_ADAPTING, bool),
+    (CONF_ALT_DETECT_METHOD, DEFAULT_ALT_DETECT_METHOD, bool),
+    (CONF_AUTORESET_CONTROL, DEFAULT_AUTORESET_CONTROL, int_between(0, 10000)),
```

Comment:
Just curious is there reason why the max limit is `10000` which is ~2.7 hours? I was hoping to set this to 3 hours, but I could imagine others may desire longer.